test/fuse/linux/fuse_base.h

```diff
@@ -0,0 +1,90 @@
+// Copyright 2020 The gVisor Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <sys/uio.h>
+
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+
+namespace gvisor {
+namespace testing {
+
+constexpr char MOUNT_POINT[] = "/mnt";
```

Comment:
According to our C++ style conventions, this should be named `kMountPoint` ([example](https://cs.opensource.google/gvisor/gvisor/+/master:test/util/test_util.h;l=217?q=lang:cc%20constexpr%20char&ss=gvisor%2Fgvisor)).

Same below.